### DISCUSSION TITLE
commands stop my server

### DISCUSSION THREAD

**Discussion by KiwiMCGaming** [EssentialsX/Essentials#4268] [Q&A] [answered]
I have loaded EssentialsX Stable and Dev builds at different times on my Spigot 1.17 Minecraft server, when I use commands /ec and /invsee My server immediately cuts out and stops. This has happened with the GSIT plug in on the /Lay plug in.
Any ideas?

**Comment by baailey** [comment, marked as answer]
There was a spigot issue where /ec and /invsee would crash the server, so that may be your issue. I recommend updating spigot/paper and then trying to recreate the problem.
